spec/support/request_helpers.rb

```diff
@@ -27,8 +27,26 @@ def api_request_headers
     expect(response.status).to eq(401)
     expect(response_json).to eq(
       {
-        "errors" => ["Not authorized, please login"]
+        "errors" => ["The access token is invalid"]
       }
     )
   end
 end
+
+RSpec.shared_examples "an endpoint that times out sessions" do |url|
+  it "returns a 401 response when the access_token has expired" do
+    expect(token).not_to be_nil, "must supply a token in the spec scope"
+
+    Timecop.travel(3.hours.from_now) do
```

Comment:
The only thing I noticed that maybe this test would be clearer if you used the settings and added minutes to it, same as at the `spec/features/users_session_timing_out_spec.rb`. But it's no biggie.